recipe/meta.yaml

```diff
@@ -12,9 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: true  # [py>=32]
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 1
+  script: "{{ PYTHON }} -m pip install . -vv"  # [py<32]
```

Comment:
> No skips except for python version. If the recipe is py3 only, remove skip statement and add version constraint on python in `host` and `run` section.

ref: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
I explicitely made it an arch dependent package. This means that downstream packages can become arch dependent.